etc/config/java.defaults.properties

```diff
@@ -0,0 +1,17 @@
+# Default settings for Java
```

Comment:
We ought to think how we can get a hold of this for the live site, but this is a great start

Reply:
Looks like @partouf has done a great job on this already!